src/HIE/Bios/Cradle.hs

```diff
@@ -466,13 +470,53 @@ cabalProcess workDir command args = do
   let newEnvironment = extraEnvironmentArgs ++ environment
   wrapper_fp <- withCabalWrapperTool ("ghc", []) workDir
   buildDir <- cabalBuildDir workDir
-  let cabalArgs = ["--builddir=" <> buildDir, command, "--with-compiler", wrapper_fp] ++ args
-  let cabalProc = proc "cabal" cabalArgs
+  let baseCabalArgs = ["--builddir=" <> buildDir, command, "--with-compiler", wrapper_fp]
+  extraCabalArgs <- case ghcDirLoadResult of
+        CradleSuccess (ghcBin, libdir) -> do
+          ghcPkgPath <- withGhcPkgTool ghcBin libdir
+          pure ["--with-hc-pkg", ghcPkgPath]
+        _ -> pure []
+  let cabalProc = proc "cabal" (baseCabalArgs ++ extraCabalArgs ++ args)
   pure $ CradleSuccess (cabalProc
       { env = Just newEnvironment
       , cwd = Just workDir
       })
 
+withGhcPkgTool :: FilePath -> FilePath -> IO FilePath
+withGhcPkgTool ghcPathAbs libdir = do
```

Comment:
Document our assumptions here and why it's different on windows, maybe

Reply:
Docs are improved now.